data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -547,19 +547,19 @@
 
         [message]
             speaker=unit
-            message= _ "Who goes there! Come out of the house now!"
+            message= _ "Who goes there! Come out now!"
```

Comment:
"Come out now!" is less stilted than the original, but "Show yourself!" might be better yet.